explainer.md

```diff
@@ -182,6 +182,50 @@ The main benefit to this approach is that it would ensure that developers don't
 
 As an aside, the `notificationclick` event has similar challenges to the `launch` event in that handlers can be written such that nothing happens when a notification is clicked. Whatever solution is decided for `launch` event should also apply to `notificationclick` for consistency.
 
+### Should apps that aren't installed have launch events fired?
+
+It is unclear whether launch events should be gated on installation. Ultimately this should be up to the user agent, but it would be nice to be able to provide some kind of recommendation. 
+
+Our preference here is that apps which are not installed don't receive launch events, so as to break user expectations of the web as little as possible (installed apps are a new thing). In general, we prefer not to gate APIs on installation, however, there is some precedence for granting more permissions upon installation; PiP and WebShareTarget, for example.
+
+### Potentially Breaking `window.open`
+`window.open` is a synchronous  API, while the nature of launch events requires them to be asynchronous. It is unclear how we will be able to reconcile these two APIs (blocking until the launch event completes doesn't seem super acceptable). This could break popup based authentication flows (and potentially more).
+
+Due to this concern, the initial implementation of launch events will likely ship without `window.open` support, with the intent to revisit it later.
+
+### Login Flows
```

Comment:
I think this section could use some reworking to make it more concise. Is this an accurate summary:
Problem: Some authentication flows redirect the user to an authentication provider's site to login before taking the user back to the original site. If the navigation to the authentication provider is captured in a launch event and sent to an app window, it will break the authentication flow. 
Solution: Apps that are authentication providers should not have special handling for login links.
[Example of how authentication providers would do this]

Reply:
Do you think we need an example here? I take it you're referring to #10?

With `launch` events, our solution is to simply not open a new window for these navigations. The manifest update (with opt out links) seems like more of an intent picker thing than a launch event thing.

I'm updating this to be more succinct (I pretty much lifted it from one of the docs). You're right, this explainer doesn't need this much detail.

Reply:
This lgtm now :)